openssa/core/programming/hierarchical/plan.py

```diff
@@ -154,11 +154,24 @@ def execute(self, knowledge: set[Knowledge] | None = None, other_results: list[A
             for sub_htp in tqdm(decomposed_htp.sub_htps):
                 sub_results.append((sub_htp.task.ask, sub_htp.execute(knowledge=knowledge, other_results=sub_results)))
 
-            inputs: str = ('REASONING WITHOUT SUPPORTING/OTHER RESULTS '
-                           '(preliminary conclusions here can be overriden by more convincing supporting/other data):\n'
-                           f'{reasoning_wo_sub_results}\n'
-                           '\n\n' +
-                           '\n\n'.join((f'SUPPORTING QUESTION/TASK #{i + 1}:\n{ask}\n'
+            # inputs: str = ('REASONING WITHOUT SUPPORTING/OTHER RESULTS '
```

Comment:
@nguyennm1024 cc @TheVinhLuong102 : Please decide whether we remove this code or we use this code in a differnt flow.